runner/src/runner/events.clj

```diff
@@ -0,0 +1,58 @@
+(ns runner.events
+  (:require [clojure.data.json :as json])
+  (:import [com.rabbitmq.stream ConfirmationHandler]
+           [com.rabbitmq.stream.impl StreamProducer]
+           [java.time Instant]))
+
+(defn publish
+  [^StreamProducer producer ^String message]
+  (let [message (.. producer
+                    messageBuilder
+                    (addData (.getBytes message))
+                    build)]
+    (.send producer message (reify ConfirmationHandler
+                              (handle [_ _])))))
```

Comment:
maybe we can add a warn here to notify if the event deliveries are failing using the [confirmation status](https://rabbitmq.github.io/rabbitmq-stream-java-client/stable/htmlsingle/#sending-messages) maybe?

Reply:
sure, oversaw that when copypasting your stuff:)